src/mono/mono/mini/mini-llvm.c

```diff
@@ -748,7 +748,7 @@ create_llvm_type_for_type (MonoLLVMModule *module, MonoClass *klass)
 		 */
 		/* SIMD types have size 16 in mono_class_value_size () */
 		if (m_class_is_simd_type (klass))
```

Comment:
Should this be replaced by `MONO_CLASS_IS_SIMD` as well?

Reply:
Don't think so, the types have the same size regardless of whenever SIMD is enabled or not.